requirements.txt

```diff
@@ -0,0 +1,13 @@
+certifi==2020.11.8
```

Comment:
We should be able to get rid of this file now that you've moved to poetry - `pyproject.toml` and `poetry.lock` should contain all of the package information now